src/cloffeine/async_cache.clj

```diff
@@ -1,41 +1,46 @@
 (ns cloffeine.async-cache
   (:refer-clojure :exclude [get])
-  (:require [cloffeine.common :as common]
-            [cloffeine.cache :as cache])
+  (:require [cloffeine.cache :as cache]
+            [cloffeine.common :as common])
   (:import [com.github.benmanes.caffeine.cache AsyncCache]))
 
 (defn make-cache
   "Create an AsyncCache. See `cloffeine.common/builder` for settings.
-  A semi-persistent mapping from keys to values. Values are automatically loaded by the cache asynchronously, and are stored in the cache until either evicted or manually invalidated.
-Implementations of this interface are expected to be thread-safe, and can be safely accessed by multiple concurrent threads."
+  A semi-persistent mapping from keys to values. Values should be manually set in
+  the cache using cloffeine.async-cache/put!.
+  Implementations of this interface are expected to be thread-safe, and can be
```

Comment:
the wording about the interface can be removed. It is kind of confusing.

Reply:
Agreed. Removed from the 4 relevant namespaces.